packages/dmg-builder/src/dmg.ts

```diff
@@ -115,15 +115,17 @@ export class DmgTarget extends Target {
     await exec("codesign", args)
   }
 
-  computeVolumeName(custom?: string | null): string {
+  computeVolumeName(arch: Arch, custom?: string | null): string {
     const appInfo = this.packager.appInfo
     const shortVersion = this.packager.platformSpecificBuildOptions.bundleShortVersion || appInfo.version
+    const archString = getArchSuffix(arch)
 
     if (custom == null) {
-      return `${appInfo.productFilename} ${shortVersion}`
+      return `${appInfo.productFilename} ${shortVersion}${archString}`
```

Comment:
For backward compatibility, I think it's better if arch suffix is only added for `arm64` builds. If we add suffix to `x64` builds, would that break `electron-updater`?

Reply:
Great callout, `getArchSuffix` actually already handles that logic:
```
export function getArchSuffix(arch: Arch): string {
  return arch === Arch.x64 ? "" : `-${Arch[arch]}`
}
```
https://github.com/electron-userland/electron-builder/blob/master/packages/builder-util/src/arch.ts#L27-L29